src/PathFind.cpp

```diff
@@ -65,7 +65,10 @@ string do_readlink(std::string const& path)
     ssize_t len = ::readlink(path.c_str(), buff, sizeof(buff)-1);
 #endif
 
-    return string(buff);
+    if ( len > 0 )
```

Comment:
Since `len` is only defined for above platforms, this line probably breaks us of this library on other platforms, e.g., win or osx.